packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -204,6 +204,16 @@ export const Stackbox = () => {
     );
   };
 
+  const handleStartDateTimeChange = (newDateTime: Date) => {
+    const dateNowInMs = Date.now();
+    const newStartDate =
+      newDateTime.getTime() <= dateNowInMs
+        ? new Date(dateNowInMs)
+        : newDateTime;
```

Comment:
suggestion: remove `dateNowInMs` variable and use `Date.now()` directly.

```suggestion
    const newStartDate =
      newDateTime.getTime() <= dateNowInMs
        ? new Date(Date.now())
        : newDateTime;
```